packages/spec-driver-selenium/src/spec-driver.ts

```diff
@@ -63,6 +63,16 @@ export function transformSelector(selector: Selector | CommonSelector): Selector
   }
   return selector
 }
+export function untransformSelector(selector: Selector): CommonSelector {
```

Comment:
I think you also need to add a passthrough logic for selectors that are already common.
```suggestion
export function untransformSelector(selector: Selector | CommonSelector): CommonSelector {
```

Reply:
Okay